doc/howto/how_to_build_and_install.md

```diff
@@ -35,10 +35,11 @@ if you want to use python API, you need to install python and swig:
 [swig office install](http://www.swig.org/download.html)
 
 ## Build YDLidar-SDK
-### Ubuntu 18.04/16.04/14.04 LTS
+### Ubuntu 22.04/18.04/16.04/14.04 LTS
```

Comment:
I have tested ubuntu 22.04.
So I added this update, but did not test ubuntu20.04.
What should we do?